src/events/messageCreate/train.ts

```diff
@@ -0,0 +1,34 @@
+import { ArgsOf, Discord, On } from "discordx";
+
+let activeTrain = false;
+let previousString = "";
+let previousUser = "";
+@Discord()
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+abstract class Train {
+	@On({ event: "messageCreate" })
+	private async handleTrain([msg]: ArgsOf<"messageCreate">) {
+		if (!activeTrain) {
+			// eslint-disable-next-line no-void
+			void msg.channel.messages.fetch({ limit: 2 }).then(async (msgs) => {
```

Comment:
Could use await here maybe instead of then
`const msgs = await msg.channel.messages.fetch({ limit: 2 });`